extensions/image-preview/src/preview.ts

```diff
@@ -105,7 +133,8 @@ class Preview extends Disposable {
 			switch (message.type) {
 				case 'size':
 					{
-						this._imageSize = message.value;
+						const { size } = fs.statSync(resource.path);
```

Comment:
Many of the images we load are not on the file system. They may be `remote` or data uris, which `fs` will not understand

Try using VS Code's file system api instead: https://github.com/microsoft/vscode/blob/623bae89051768933b9a14e93a2e9b496526adfb/src/vs/vscode.d.ts#L5976